src/experimental-bots/filler/fillerMultithreaded.ts

```diff
@@ -598,9 +655,16 @@ export class FillerMultithreaded {
 	public healthCheck(): boolean {
 		if (!this.dlobHealthy) {
```

Comment:
i think it might better to increment this every time `healthCheck` is called (so you can see the counter going up as another check) otherwise the series wont exist until the first heartbeat is missed, so i'd replace the 2 heartbeat counters you added with:
```suggestion
		if (!this.dlobHealthy) {
			this.heartbeatCounter?.add(1, {
				marketIndexes: this.config.marketIndex,
				type: 'dlobBuilder',
				success: 'false'
			});
		} else {
			this.heartbeatCounter?.add(1, {
				marketIndexes: this.config.marketIndex,
				type: 'dlobBuilder',
				success: 'true'
			});
		}
```